src/tests/init.rs

```diff
@@ -16,7 +16,7 @@ fn test_init_token() {
     let mut args = TestToken::make_init_args(Some(testtokn.make_init_string()));
     let args_ptr = &mut args as *mut CK_C_INITIALIZE_ARGS;
     let mut ret = fn_initialize(args_ptr as *mut std::ffi::c_void);
-    assert_eq!(ret, CKR_OK);
+    assert!(ret == CKR_OK || ret == CKR_CRYPTOKI_ALREADY_INITIALIZED);
```

Comment:
wonder if something like this would help the reporting and would be more "rust-way":
```suggestion
    match ret {
        CK_OK | CKR_CRYPTOKI_ALREADY_INITIALIZED => (), // OK
        _ => panic!("C_Initialize returned unexpected value {ret}")
    }
```